src/applications/travel-pay/containers/TravelPayStatusApp.jsx

```diff
@@ -88,18 +88,19 @@ export default function App({ children }) {
   }
 
   // TODO: Move this logic to the API-side
+  const compareClaimsDate = (a, b) => {
+    return (
+      Date.parse(b.appointmentDateTime) - Date.parse(a.appointmentDateTime) ||
+      Date.parse(b.createdOn) - Date.parse(a.createdOn)
```

Comment:
It took me a while to figure out that this works because doing a `Date.parse` of a null value returns `NaN`, which is falsy. Maybe a comment here would go down smooth.

Reply:
Good call, added some clarification in https://github.com/department-of-veterans-affairs/vets-website/pull/31348/commits/e8bdbff48aa28a0a73cfbc4cc9df6e32aad6fbaf